content/php/concepts/math-functions/terms/decbin/decbin.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'decbin()'
+Description: 'Converts a decimal number to a binary number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+  - 'Math Functions'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`decbin()`** function converts a decimal number to a binary number.
+
+The required parameter is the `number` that is used to specify the decimal value to convert. The result returns a `string` that contains the binary number of the decimal value.
+
+## Syntax
+
+```pseudo
+decbin(number); 
+
+```
+
+Where the `number` is the decimal value to be converted to binary.
+
+## Example
+
+```php
+echo decbin(333)
+// Output: 101001101
+```
+
+The result will be the binary number `101001101`.
+
+## Codebyte Example
```

Comment:
I would recommend removing the codebyte since it isn't demonstrating any new functionality beyond the previous example. We aim not to duplicate examples. Each example should have a purpose.

Reply:
noted on this

thanks